managed/services/supervisord/logs.go

```diff
@@ -360,3 +372,27 @@ func addAdminSummary(ctx context.Context, zw *zip.Writer) error {
 
 	return nil
 }
+
+func maskSensitiveValues(data []byte) ([]byte, error) {
+	var maskedValue = "**********"
+	var c alertmanager.Config
+	err := yaml.Unmarshal(data, &c)
+	if err != nil {
+		return data, err
+	}
+	c.Global.SMTPAuthUsername = maskedValue
+	c.Global.SMTPAuthPassword = maskedValue
+	c.Global.SMTPAuthSecret = maskedValue
+	c.Global.SMTPAuthIdentity = maskedValue
+	c.Global.SlackAPIURL = maskedValue
+	c.Global.PagerdutyURL = maskedValue
+	c.Global.OpsGenieAPIURL = maskedValue
+	c.Global.OpsGenieAPIKey = maskedValue
+	c.Global.WeChatAPIURL = maskedValue
+	c.Global.WeChatAPISecret = maskedValue
+	c.Global.WeChatAPICorpID = maskedValue
+	c.Global.VictorOpsAPIURL = maskedValue
+	c.Global.VictorOpsAPIKey = maskedValue
+
+	return nil, nil
```

Comment:
I guess we should marshal config back to bytes and retrun it here.

Reply:
Looks like we could have a custom `UnarshalYAML` implementation and do the masking there too.